erpnext/projects/doctype/project/project.json

```diff
@@ -121,6 +127,35 @@
    "allow_on_submit": 0, 
    "bold": 0, 
    "collapsible": 0, 
+   "columns": 0, 
+   "default": "Task Completion", 
+   "fieldname": "pct_comp_method", 
```

Comment:
1. I have made the changes regarding the renaming of the field to percent_complete_method.

1. For the % Complete Method and %Tasks Completed, I feel they need to remain on the top where the user can see them without scrolling. The reasons being that %Complete Method usually is set up when the project is set up initially (you may or may not know all tasks at this point) and once the project starts a key parameter is %Tasks Completed which should be available upfront without scrolling.

Also both the fields with a vertical scroll in between may mess up the Grid alignment.

You can let me know if you still want #2 to be changed and I will re-push the changes accordingly.